client/src/components/createLanguageRedirect.test.js

```diff
@@ -59,6 +64,16 @@ describe('createLanguageRedirect for clientLocale === english', () => {
       });
       expect(receivedPageURL).toBe(francaisPageURL);
     });
+
+    it('should not replace the second espanol', () => {
```

Comment:
If we have ever had such a use case, then we would need to rethink it instead.

Reply:
Agreed. I get that this is an edge case, but I figured it was easy to prevent.

Reply:
I think this change needs to be reverted, because we know we do not have such a case as of now. When we do have a language segment in the path we need to think the "why" we have the case. 

It needs be caught (most likely by a user) as eitherways (prevent it now or not). I would much rather not fix it yet.